### PR TITLE
Adding limitations on team-handle characters

### PR DESCRIPTION
**Pull request by DavidGrashton**
Differentiating between the rules for team-handle vs team-name characters

### What does this PR do? What is the motivation?

Customer reported confusion based on the documentation, which listed the same rules for team names and team handles, but team handles actually have more restrictions

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

Testing in the UI, it seems like the team name actually allows spaces and a greater range of special characters. Not sure if we want to update the doc to reflect this, or if there is some other reason why only the few characters allowed in team handles should be recommended there.

### CODE REVIEW

File: content/en/account_management/teams/_index.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Access the team directory page from [Organization Settings][1] or by navigating
 1. Provide an optional **Description**.
 1. Click **Create**.
 
-**Note:** Allowed characters for team names and team handles are `a-z`, `A-Z`, `0-9`, and `._-:/`. Replace spaces with underscores.
+**Note:** Allowed characters for team names are `a-z`, `A-Z`, `0-9`, and `._-:/`. Replace spaces with underscores. Allowed characters for team handles are `a-z`, `0-9`, and `._-:/`, and the last character cannot be an underscore.
```
Comment (drichards-87) [inline content/en/account_management/teams/_index.md:27]:
```suggestion
**Notes**: 

- Allowed characters for team names are `a-z`, `A-Z`, `0-9`, and `._-:/`. Replace spaces with underscores. 
- Allowed characters for team handles are `a-z`, `0-9`, and `._-:/`. The last character cannot be an underscore.
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
@DavidGrashton I made a suggestion on changing the format into a bulleted list. Thoughts?

File: content/en/account_management/teams/_index.md (hunk 1 above)
Comment (DavidGrashton) [reply content/en/account_management/teams/_index.md:27]: Sounds great!

**Comment by drichards-87** [comment]
@DavidGrashton Just wanted to confirm that this is ready to be merged now.

**Comment by DavidGrashton** [comment]
Hi @drichards-87, I believe it is ready to merge but I'm not sure why the Preview Synthetics check is timing out

**Comment by drichards-87** [comment]
@DavidGrashton Sounds good. I'll get the PR merged now. As far as the check failing, it's because the branch name is missing a forward slash. We require a forward slash in order to generate a staging preview of the docs site. This requirement also has the side effect of not allowing the build job to fully pass when a branch is missing the forward slash. In this case, I already pulled your branch down locally and tested the changes myself to confirm everything looks as expected.

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**